src/librustc_session/lint.rs

```diff
@@ -195,7 +195,7 @@ pub enum BuiltinLintDiagnostics {
 }
 
 /// Lints that are buffered up early on in the `Session` before the
-/// `LintLevels` is calculated. These are later passed to `librustc`.
+/// `LintLevels` is calculated. These are later passed to `librustc_middle`.
```

Comment:
This is the other "passed to" (GH won't let me do multi-line comments on this diff).